core/src/main/scala/io/projectglow/bgen/BgenFileFormat.scala

```diff
@@ -38,6 +38,7 @@ import org.skife.jdbi.v2.util.LongMapper
 import io.projectglow.common.logging.{HlsMetricDefinitions, HlsTagDefinitions, HlsTagValues, HlsUsageLogging}
 import io.projectglow.common.{GlowLogging, WithUtils}
 import io.projectglow.sql.util.{ComDatabricksDataSource, SerializableConfiguration}
+import io.projectglow.vcf.VCFOption
```

Comment:
It may be a good idea to gather all bgen options (including this one) in a BGENOption object like what we did for VCFOption. It will be cleaner and creates separation of VCF and BGEN. Just a thought. Let me know if you disagree and I will approve.